### DISCUSSION TITLE
ФРМО. Организация. Тип субъекта системы здравоохранения (строка файла 151, 157)

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#65] [Административная РГ]
Что это за информация? Где посмотреть справочник?

**Comment by IvanekOS** [comment]
> Что это за информация? Где посмотреть справочник?

Добрый день!
Допустимые значения для описания типов субъекта системы здравоохранения, к которому относится организация:
• Медицинская организация;
• Организация здравоохранения;
• Фармацевтическая организация;
• Образовательная организация

Дополнительно обсуждается вопрос добавления лицензируемых организаций, но этот момент требует последующего уточнения в части использования.
